test/user.test.js

```diff
@@ -1807,9 +1809,10 @@ describe('User', function() {
 
   describe('Email Update', function() {
     describe('User changing email property', function() {
-      var user;
+      var user, originalUserToken1, originalUserToken2, newUserCreated, userSpecial, userNormal;
```

Comment:
AFAICT, `userSpecial` and `userNormal` are used only in a single test only. Could you please move the declaration of these two variables down to the test case?